packages/forms-web-app/src/dynamic-forms/validator/requiredValidator.test.js

```diff
@@ -0,0 +1,11 @@
+const RequiredValidator = require('./requiredValidator');
+describe('./src/dynamic-forms/validator/booleanValidator.test.js', () => {
+	it('should return an error message if the boolean field is missing', () => {
```

Comment:
It's probably worth covering off the opposite scenario too to ensure we don't get a validation error when the value is valid.